config/core/test-configs.yaml

```diff
@@ -2473,6 +2490,7 @@ test_configs:
       - cros-ec
       - igt-gpu-panfrost
       - igt-kms-rockchip
+      - kselftest-alsa
```

Comment:
We'll see how this goes, if it adds too much load in production we might have to drop it.

Reply:
Failed to boot: https://staging.kernelci.org/test/plan/id/6259450bd7075bb35505f4e6/

I suggest we drop this one too.